### DISCUSSION TITLE
What does "We have X sample image(s)." mean?

### DISCUSSION THREAD

**Discussion by NoteToSelfFindGoodNickname** [d8ahazard/sd_dreambooth_extension#791] [Q&A]
I am fine-tuning a model, and I use 28 images.
When the training finished, the console said: "We have 3 sample image(s)."

I wonder what this means. 

Here is the full log:

![ximages](https://user-images.githubusercontent.com/29504639/212908370-f9ee776e-554f-4451-a0d0-3c865e8c04c2.png)

**Comment by NoteToSelfFindGoodNickname** [comment]
Bump

**Reply by NoteToSelfFindGoodNickname** [reply]
Bumpidibump.

**Reply by Zuxier** [reply]
1 sample image and 2 graphs
